src/fake-timers-src.js

```diff
@@ -1,6 +1,13 @@
 "use strict";
 
-const globalObject = require("@sinonjs/commons").global;
+// Node.js stores certain things like the performance object on globalThis
+// but not on its `global` object which means we prefer it to `global` in
+// node versions that support it - diverging from the @sinonjs/commons behavior
+/* global globalThis */
```

Comment:
This is all fine and dandy, but is there a reason we should not include this change in `@sinonjs/commons`?

Reply:
Well, it changes the meaning of what `global` is and I am not sure that applies to other sinon projects. Let me take another look at the code.

Reply:
I'm trying to figure out why performance won't be on `global`